### DISCUSSION TITLE
Recording 24/7 but with less FPS (Transcoding -> Transrating)

### DISCUSSION THREAD

**Discussion by Yszty** [blakeblackshear/frigate#8734] [Ask A Question]
Hello, 
I want record 24/7 but only with 3 FPS. Its possible to configure?
Best regards

**Comment by NickM-27** [comment]
The best way to do this is configure on the camera side

**Reply by Yszty** [reply]
But, I want long storage (1 year from 6 cameras) with lower FPS and high resolution and live view on demand with high FPS and high resolution. Substream doesnt resolve this problem becouse he allow only lower resolution. 
Maby go2rtc can resolve this?

**Reply by moandcompany** [reply]
You can accomplish this by using go2rtc and passing arguments to ffmpeg to transcode the camera's original higher-resolution, high-frame rate stream into whatever framerate, bitrate you want.
